nanobinds/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# ---------------------------------------------------------------------
+# Create a single python module named _ompl
+# ---------------------------------------------------------------------
+file(GLOB_RECURSE PY_OMPL_SOURCES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} base/*.cpp control/*.cpp geometric/*.cpp util/*.cpp python.cpp)
+nanobind_add_module(_ompl
+    STABLE_ABI
+    ${PY_OMPL_SOURCES}
+)
+
+if (DEFINED CMAKE_LIBRARY_OUTPUT_DIRECTORY)
+    set_target_properties(_ompl PROPERTIES LIBRARY_OUTPUT_DIRECTORY
+        "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/ompl")
+else()
+    set_target_properties(_ompl PROPERTIES LIBRARY_OUTPUT_DIRECTORY "ompl")
+endif()
+
+target_link_libraries(_ompl PRIVATE ompl::ompl)
+target_compile_options(_ompl PRIVATE -Wno-unused-parameter)
+
+# Set RPATH so the module can find libompl.so at runtime during development
+set_target_properties(_ompl PROPERTIES
+    BUILD_RPATH "$<TARGET_FILE_DIR:ompl>"
+    INSTALL_RPATH "$ORIGIN;$ORIGIN/../lib"
+)
```

Comment:
todo: reexamine if this is needed, ideally we don't interfere with user's RPATHs. Same with the library output dir